pkg/logs/config/config.go

```diff
@@ -176,51 +176,86 @@ func buildTCPEndpoints() (*Endpoints, error) {
 	return NewEndpoints(main, additionals, useProto, false, 0), nil
 }
 
+// LogsConfigKeys stores logs configuration keys stored in YAML configuration files
+type LogsConfigKeys struct {
+	UseCompression          string
+	CompressionLevel        string
+	ConnectionResetInterval string
+	LogsDDURL               string
+	LogsNoSSL               string
+	DDURL                   string
+	DevModeNoSSL            string
+	AdditionalEndpoints     string
+	BatchWait               string
+}
```

Comment:
Maybe we can add also the possibility to provide the ApiKey that is different from the global setting.

Reply:
We currently don't have this use case for our two new endpoints. However, we could consider this is a following PR.